.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -20,16 +20,12 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop/Laptop (please complete the following information):**
- - OS: [e.g. MacOS 10.13.3]
- - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser: [e.g. stock browser, safari]
- - Version: [e.g. 22]
+**Environment information**
+ - aut version: [e.g. 0.16.0, HEAD]

```

Comment:
We tend to capitalize `aut` as `AUT` throughout our docs, could we change it here for consistency?